tests/utils/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './utils';
+export * as exec from './exec';
+export * as platform from './platform';
```

Comment:
I suggest that these are all:

```
export * from './exec';
export * from './platform';
```

This matches how `src/utils/index.ts` does it.

Reply:
Only need to namespace here if the functions will conflict.